docs/FRSKY_R9.md

```diff
@@ -70,31 +88,94 @@ The wiring is similar to the ExpressLRS docs found in the link below. Different
 The R9MX receiver can also be used as mLRS Tx module. Due to hardware restrictions it is somewhat limited but can be a great choice for special applications. 
 
 The module can be configured via CLI only. The CLI shares the same port used for serial/Mavlink communication. Switching between CLI and serial port is done via the bind button:
-- Powering up the R9MX without holding down the bind button will boot in "Serial" mode. To enter CLI mode, press the bind button while powering up. The device then runs in CLI mode, and the Rx and Tx can be configured as usual via CLI commands.
+- Powering up the R9MX without holding down the bind button will boot in "Serial" mode. To enter CLI mode, press the bind button while powering up or shortly after powering up if using the ELRS bootloader. The device then runs in CLI mode, and can be configured as usual via CLI commands.
+
+The wiring is similar to when used as a receiver, with the exception that the "Inverted SPort" pin (original naming) is available as input port.
+
+
+## R9MM Receiver ##
 
-The wiring is similar to Rx and ExpressLRS, with the exception that the "Inverted SPort" pin (orginal naming) is available as input port.
+The R9MM receiver is also supported. However, flashing the R9MM with ST-Link is tedious and requires top soldering skills or employing other tricks, as one needs to connect to four tiny solder pads in a tight space. If you want to flash via ST-Link, please consider using the R9MX receiver instead.
 
+If you want to use ELRS bootloader and install via the Frsky bootloader and OpenTX or EdgeTX, the R9MM is the smallest and lightest receiver available.  Note: The R9MM has a different [wiring diagram](https://www.expresslrs.org/quick-start/receivers/r9/#wiring-up-your-receiver) than the R9MX.
 
 ## Flashing ##
 
-ExpressLRS has figured out a convenient and easy way for flashing, which is unfortunately not available for mLRS (someone needs to figure it out). For flashing mLRS only the grass-route DIY procedure using a ST-Link programmer is currently possible.
+### Acknowledgments ###
```

Comment:
you guys decide against some intro text before this first sub chapter?
If so I'm sure fine with it, just wanted to ask :)